os/drivers/task_manager/task_manager_drv.c

```diff
@@ -161,6 +161,7 @@ static int taskmgr_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 			return ERROR;
 		}
 		(void)sigprocmask(SIG_SETMASK, NULL, &tcb->sigprocmask);
+		ret = OK;
```

Comment:
Between 207 and 207 lines, break is required.

Reply:
fixed